src/ResizeSensor/index.ts

```diff
@@ -0,0 +1,3 @@
+import ResizeSensor, { ResizeSensorProps, Props } from './ResizeSensor';
```

Comment:
`Props` is never exported by `index.ts` (no real reason, just to be consistent with other components)